spring-data-jdbc/src/test/java/org/springframework/data/jdbc/repository/JdbcRepositoryIdGenerationIntegrationTests.java

```diff
@@ -107,15 +131,126 @@ void manuallyGeneratedIdForSaveAll() {
 		assertThat(immutableWithManualIdEntityRepository.findAll()).hasSize(2);
 	}
 
+	@Test // DATAJDBC-2003
+    @Sql(statements = "INSERT INTO EntityWithSeq(id, name) VALUES(1, 'initial value');")
+	void testUpdateAggregateWithSequence() {
+
+        // given.
```

Comment:
Can we omit these inline comments? Having domain classes designed for single-line instantiation along with values makes the code more compact and increased information density instead of spreading everything out to individual lines.

Reply:
@mp911de  Agree, locality is important, I'll remove the `given`/`when`/`then`. What are your thoughts about other inline comments? I'm talking about this:

```
        // We do not expect the SELECT next value from sequence in case we're doing an INSERT with ID provided by the client
        auditableNamedParameterJdbcTemplate.assertSequenceOfStatements(INSERT);
```

I think it makes sense to retain them.

Reply:
I wonder how much a P6Spy-based approach would help us as that is encapsulating the actual connection with the ability to get hold of all statements.

Reply:
Closed it to retain discussion in one place: https://github.com/spring-projects/spring-data-relational/pull/2005#issuecomment-2736140055